draft-eggert-bcp45bis.md

```diff
@@ -178,6 +183,8 @@ pattern of abuse. They are encouraged to take into account the overall nature
 of the postings by an individual and whether particular postings are an
 aberration or typical.
 
+**RW: Should it be possible to appeal the IETF chairs appointment of an SAA individual, i.e., via the same process as below?  I presume so (like any decision), but would it be helpful for this to be stated?**
+
```

Comment:
Not sure if something needs to be said. Appointing an SAA is an AD action that can be appealed like any other AD action?

Reply:
I'm okay either way on this.  I was just thinking of Keith's comments that he wanted the SAA to be more decoupled from the IETF chair and IESG ... which it seems to be.

Reply:
SAA is completely decoupled from the IESG. Only the chair has direction over them, and that has pretty much been relinquished to them.

Reply:
Okay.